### ISSUE TITLE
Long table got cut off in the pdf version of ECMA402

### ISSUE THREAD

**Issue by FrankYFTang** [tc39/ecma402#561]
I just spot this issue int he Jun 2020 version of ECMA402
notice in page 15 of 
https://www.ecma-international.org/wp-content/uploads/ECMA-402.pdf

The content of "Table 2: Simple units sanctioned for use in ECMAScript" is cut off after "ounce" and the following units are missing from the standard (but it should be there)

percent
petabyte
pound
second
stone
terabit
terabyte
week
yard
year

@leobalter @sffc @gibson042

**Comment by FrankYFTang** [comment]
We should be careful of 2021 version. We should also pay attention to the table of Numbering Systems. It does not get cut off in the last version but is also very long and may got cut off this year. Please review about the table issue carefully this year.

**Comment by leobalter** [comment]
Yes, the process of making this printable version has historically being painful. Tables are the first items to break, and usually in the horizontal aspect instead. Thanks for catching that up.

**Comment by sffc** [comment]
What is the status on this?

**Comment by leobalter** [comment]
@sffc not yet done, I needed to wait until the current TC39 due to the pile of work. 

I need to have a good printed version that fits the table and sync with the frontmatter using the registered Adobe tools. In the last time @ljharb helped me with the former.
